codechecker_lib/analyzers/analyzer_types.py

```diff
@@ -347,6 +349,21 @@ def construct_result_handler(args,
                 buildaction,
                 report_output,
                 lock)
+    elif export_plist_path:
+        if buildaction.analyzer_type == CLANG_SA:
```

Comment:
Note to a future refactoring: it might scale better to have a seperate post processing step to plist and the next stage would consume the plist regardless what happened before, so no branching on the analysis type is required here.